.github/workflows/nightly_detect_failure.yml

```diff
@@ -44,7 +44,14 @@ jobs:
         toolchain=$(<lean-toolchain)
```

Comment:
Out of reach for commenting, but two lines up it still says “branch”, not “tag”.

Reply:
Fixed, thanks.